gtsam/hybrid/GaussianMixture.h

```diff
@@ -174,20 +174,44 @@ class GTSAM_EXPORT GaussianMixture
       const VectorValues &continuousValues) const;
 
   /**
-   * @brief Compute the logProbability of this Gaussian Mixture given the
-   * continuous values and a discrete assignment.
+   * @brief Compute the error of this Gaussian Mixture.
+   *
+   * This requires some care, as different mixture components may have
+   * different normalization constants. Let's consider p(x|y,m), where m is
+   * discrete. We need the error to satisfy the invariant:
+   *
+   *    error(x;y,m) = K - log(probability(x;y,m))
+   *
+   * For all x,y,m. But note that K, for the GaussianMixture, cannot depend on
```

Comment:
I am not sure I follow the reasoning for this (other than the fact that the K is the inverse of the sum of all the conditionals by definition). Can you please elaborate?

Reply:
I think you reviewed and read comment in `Conditional.h`? Is it not clear in light of that?

Reply:
I think it should say "K should not depend" and not "K cannot depend", since the "cannot" contradicts the statement in `Conditionals.h` about overriding.